### ISSUE TITLE
windows 10 22h2 compatibilty

### ISSUE THREAD

**Issue by asherrove18** [niivu/Windows-11-themes#10]
Does it works properly with windows 10 22h2?

**Comment by DDroid** [comment]
> Does it works properly with windows 10 22h2?

No. These themes are only compatible with Windows 10 and 11 version 21H2, at the newest. If you're looking for themes compatible with Windows 10/11 22H2, you will need to look elsewhere. 

Sorry.

**Comment by Paradoxum** [comment]
https://www.deviantart.com/niivu/art/NORD-for-Windows-11-22H2-928730071

**Comment by DDroid** [comment]
> https://www.deviantart.com/niivu/art/NORD-for-Windows-11-22H2-928730071

As far as I know, some of those themes (I tested with a few, including "Windows 11 Redux") didn't work or apply on newer versions of Windows 11 22H2 when using SecureUXTheme. Simply nothing applied, aside from the wallpaper. I'm going to go out on a limb and say, perhaps later version of Windows 11 22H2 aren't compatible.

**Comment by Paradoxum** [comment]
I'm using it fine on version 22621.1835

![nord 22h2](https://github.com/niivu/Windows-11-themes/assets/6451395/cbb6d67a-6a72-4da7-8e2f-81f4bca80763)

The only issue is that sometimes the right click context menu for things messes up:
![image](https://github.com/niivu/Windows-11-themes/assets/6451395/f99f49a5-984b-43d8-8495-06c766b79914)

Sometimes you can't even click anything. Can't figure out what triggers it, happens seemingly at random to random programs.

**Comment by DDroid** [comment]
> I'm using it fine on version 22621.1835

Likely there is an incompatibility with the version I run then, (22621.1848). Despite following all directions, none of the 22H2 themes by this author work for me, or at least the 22H2 Windows 11 themes I was interested in using. The other themes look all too copy and paste for my interest, and a bit too MacOS styled. I was looking more for the Aero Styling of the Windows 11 Redux 22H2 theme in particular, but I did try a few others to see if they would apply, nothing.

**Comment by Paradoxum** [comment]
Okay, here's the theme files I'm currently using on 22621.1848
https://atomicrhino.net/upload/Using_Themes_21-06-23.7z
'Nordwind' is the one I modified to fix the text colour in the navbar. The other two are stock Nord.

I patch the system with ThemeTool
![image](https://github.com/niivu/Windows-11-themes/assets/6451395/26cfe6ba-d000-4c31-adab-b26fbfd5585d)
And just use install Rename DefaultColours.

I figured out how to fix the inability to click on some things - it's something to do with how apps are skinning themselves vs how windows is triying to skin them. What I need to do is go to the .exe and set the compatability settings to Vista SP2, it runs fine and you get this on the systray or UI of whatever it is:
![image](https://github.com/niivu/Windows-11-themes/assets/6451395/1350fea5-d19b-4f72-aaf0-b167e766db3c)


![image](https://github.com/niivu/Windows-11-themes/assets/6451395/b00b8b1a-bcb8-4dc9-9a93-d99d2a62b6d7)


Everything works but at least you can edit and touch things. It only seems to need to be done with specific tools etc, eg:
![image](https://github.com/niivu/Windows-11-themes/assets/6451395/ec15c678-63dd-4b14-ae63-4bf760b3b251)

Dirscord runs fine.
![image](https://github.com/niivu/Windows-11-themes/assets/6451395/45cd9b5a-4a9e-4dc6-91aa-4e415b6399b5)

Battl.net

Etc.

No idea why.

Just posting as much info as I can for posterity.

Thanks guys,.
